saleor/static/dashboard-next/components/Tab/SingleTab.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from "react";
+import { withStyles } from "@material-ui/core/styles";
+import Typography from "@material-ui/core/Typography";
+import * as classNames from "classnames";
+
+interface TabProps {
+  isActive: boolean;
+  value: number;
+  changeTab: (index: number) => void;
+}
+
+const decorate = withStyles(theme => ({
+  active: {},
+  root: {
+    "&$active": {
+      borderBottomColor: theme.palette.primary.main
+    },
+    borderBottomWidth: 1,
+    borderBottomStyle: "solid" as "solid",
+    borderBottomColor: "transparent",
```

Comment:
Border definition can be collapsed to one `borderBottom: "1px solid transparent"`